ui/src/app/machines/views/MachineList/OwnerColumn/OwnerColumn.js

```diff
@@ -0,0 +1,33 @@
+import { useSelector } from "react-redux";
+import React from "react";
+import PropTypes from "prop-types";
+
+import { machine as machineSelectors } from "app/base/selectors";
+
+const OwnerColumn = ({ systemId }) => {
+  const machine = useSelector(state =>
+    machineSelectors.getBySystemId(state, systemId)
+  );
+
+  const owner = machine.owner ? machine.owner : "-";
+  const tags = machine.tags ? machine.tags.join(", ") : "";
+
+  return (
+    <>
+      <div className="p-double-row__primary-row">
+        <span data-test="owner">{owner}</span>
+      </div>
+      <div className="p-double-row__secondary-row">
+        <span className="u-text-overflow" title={tags} data-test="tags">
```

Comment:
`u-text-overflow` was a custom utility in the angular version, which truncates text that overflows to a second line. I think it would be clearer if it was renamed `u-truncate-text` or something. Anyway the code for it is:

``` scss
.u-truncate-text {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}
```

IMO it should be on the parent of the text as its the parent that defines what space the text can occupy.